manifest.json

```diff
@@ -6,8 +6,9 @@
         "en": "Duniter client to manage wallets, certifications in a libre money.",
         "fr": "Client Duniter pour gérer ses portes-monnaie, les certifications au sein d’une monnaie libre."
     },
+    "version": "1.3.11~ynh1",
     "url": "https://github.com/duniter/cesium",
-    "license": "free",
+    "license": "GNU AGPL-3.0",
```

Comment:
You should strictly follow the [rule from the documentation about this field](https://yunohost.org/#/packaging_apps_manifest_en).

Reply:
Ok so I should remove GNU to respect the value from the Identifier column from https://spdx.org/licenses/ .

Reply:
You are using a deprecated licence string that can be found at the bottom of the website.

So, you have the choice between `AGPL-3.0-only` and `AGPL-3.0-or-later`.
I can tell you that this is `AGPLv3+` as Cesium license contains `later` statements.

Reply:
Sorry. I changed to `AGPL-3.0-or-later` then